weasyl/controllers/settings.py

```diff
@@ -411,6 +411,25 @@ def control_apikeys_post_(request):
 
 
 @login_required
+def control_editsearchtagblacklist_get_(request):
+    # Get the user's blacklist tag settings and render to the template
+    return Response(define.webpage(request.userid, "control/edit_searchtagblacklist.html", [
+        searchtag.get_user_searchtag_blacklist(request.userid),
+    ]))
+
+
+@login_required
+@token_checked
+def control_editsearchtagblacklist_post_(request):
+    form = request.web_input(tags="")
+    tags = searchtag.parse_blacklist_tags(form.tags)
```

Comment:
It’s probably good to start making use of Pyramid’s request properties for new code; try `request.params["tags"]` here.